Monika After Story/game/zz_backup.rpy

```diff
@@ -25,7 +25,8 @@ python early:
         import shutil
         global mas_corrupted_per, mas_no_backups_found, mas_backup_copy_failed
         global mas_backup_copy_filename, mas_bad_backups
-        early_log_path = os.path.normcase(renpy.config.basedir + "/early.log")
+
+        early_log = store.mas_logging.init_log("early", append=False)
```

Comment:
This one shouldn't be cleared every time. I had a few cases where the fact it kept the old records helped me a lot. It's important since it helps understand what happened to persistent.

Also the header is kinda redundant for this log. Would be better if we could delay log file and not print the header. This also applies to submod and bg log, we print there even with no submods installed/bg errors.